keyboards/tada68/rules.mk

```diff
@@ -1,61 +1,26 @@
 # MCU name
 MCU = atmega32u4
 
-# Processor frequency.
-#     This will define a symbol, F_CPU, in all source code files equal to the
-#     processor frequency in Hz. You can then use this symbol in your source code to
-#     calculate timings. Do NOT tack on a 'UL' at the end, this will be done
-#     automatically to create a 32-bit value in your source code.
-#
-#     This will be an integer division of F_USB below, as it is sourced by
-#     F_USB after it has run through any CPU prescalers. Note that this value
-#     does not *change* the processor frequency - it should merely be updated to
-#     reflect the processor speed set externally so that the code can use accurate
-#     software delays.
-F_CPU = 16000000
-
-
-#
-# LUFA specific
-#
-# Target architecture (see library "Board Types" documentation).
-ARCH = AVR8
-
-# Input clock frequency.
-#     This will define a symbol, F_USB, in all source code files equal to the
-#     input clock frequency (before any prescaling is performed) in Hz. This value may
-#     differ from F_CPU if prescaling is used on the latter, and is required as the
-#     raw input clock is fed directly to the PLL sections of the AVR for high speed
-#     clock generation for the USB and other AVR subsections. Do NOT tack on a 'UL'
-#     at the end, this will be done automatically to create a 32-bit value in your
-#     source code.
-#
-#     If no clock division is performed on the input clock inside the AVR (via the
-#     CPU clock adjust registers or the clock division fuses), this will be equal to F_CPU.
-F_USB = $(F_CPU)
-
-# Interrupt driven control endpoint task(+60)
-OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
-
-
-# Boot Section Size in *bytes*
-#   Teensy halfKay   512
-#   Teensy++ halfKay 1024
-#   Atmel DFU loader 4096
-#   LUFA bootloader  4096
-#   USBaspLoader     2048
-OPT_DEFS += -DBOOTLOADER_SIZE=4096
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = lufa-ms
```

Comment:
Is this even a thing?  (yet)? 

Looking at bootloader.mk, it's not.  So, it may not get the correct firmware size.

Reply:
Correct, it doesn't exist, and indeed `BOOTLOADER_SIZE` will not be defined, however as a fallback bootloader.c also contains some code that attempts to determine this by looking at the fuse bits:

https://github.com/qmk/qmk_firmware/blob/c7a8cab8831f18778e4d46e3c6bfe36803656986/tmk_core/common/avr/bootloader.c#L81-L93

Reply:
The problem is that only affects the firmware code, it doesn't affect the compile time size check, which is my concern.

Reply:
How about `atmel-dfu # actually lufa-ms`? I think I did that before for another board with a mass storage bootloader.
These ones are tricky since they steal space outside of the bootloader section, so the size check may be off anyway.

Reply:
Yeah, that should be fine, I think.  That or just leave the bootloader size but remove the bootloader (or comment it out)